docs/mutatingwebhookconfiguration.md

```diff
@@ -0,0 +1,7 @@
+# MutatingWebhookConfiguration Metrics
+
+| Metric name| Metric type | Labels/tags | Status |
+| ---------- | ----------- | ----------- | ----------- |
+| kube_mutatingwebhookconfiguration_info | Gauge | `mutatingwebhookconfiguration`=&lt;mutatingwebhookconfiguration-name&gt; <br> `namespace`=&lt;mutatingwebhookconfiguration-namespace&gt; | STABLE |
```

Comment:
New metrics have to be EXPERIMENTAL. Apologies for not being clear. We are yet to add documentation to outline this.
```suggestion
| kube_mutatingwebhookconfiguration_info | Gauge | `mutatingwebhookconfiguration`=&lt;mutatingwebhookconfiguration-name&gt; <br> `namespace`=&lt;mutatingwebhookconfiguration-namespace&gt; | EXPERIMENTAL |
```